xbmc/video/windows/GUIWindowVideoNav.cpp

```diff
@@ -268,6 +268,114 @@ std::string CGUIWindowVideoNav::GetQuickpathName(const std::string& strPath) con
   }
 }
 
+SELECT_FIRST_UNWATCHED_ITEM CGUIWindowVideoNav::SelectFirstUnwatchedItem()
```

Comment:
or BTW change the method name, it sound not like a getter for the setting option